src/reps/grip.js

```diff
@@ -243,4 +243,6 @@ let Grip = {
 };
 
 // Exports from this module
-module.exports = Grip;
+module.exports = {
+  Grip: Grip
```

Comment:
Is there a good reason to do that ? I can see in https://github.com/devtools-html/devtools-reps/pull/57/files#diff-2b328e8117f0ee91e52fa1d20eb56e3aR36 that it makes us change how we import the Grip, which now differs from the way we load the other reps : 

```javascript
const GripMap = require("./grip-map");
const { Grip } = require("./grip");
```
I think it would be better if all the import looks the same.